packages/perseus/src/__testdata__/article-renderer.testdata.ts

```diff
@@ -58,6 +58,34 @@ export const passageArticle: PerseusRenderer = {
     },
 };
 
+export const exerciseArticle: PerseusRenderer = {
```

Comment:
There's articles ... and there's exercises... and then there's exerciseArticles? 😂 

In regular articles, there is no way to "check answer" in typical usage. You'd need to nest this `expression` inside a `GradedGroup` widget in order for this to work within an Article.  I guess this story just allows you to open the keypad in the ArticleRenderer... maybe that's ok.  I guess all this text is to say I'd rather not invoke the term "exercise" in the naming.  To me that's conflicting and confusing.

Reply:
`articleWithExpression`? Or is even that too much?